docs/_includes/drawer-dialog.html

```diff
@@ -1,7 +1,7 @@
 <div id="drawer-dialog">
     {% include section-header.html name="drawer-dialog" version=page.module_metadata.drawer-dialog.ds-component.version %}
 
-    <p>** DEPRECATED ** Use <span class="highlight">lightbox-dialog</span> instead </p>
+    <p>** DEPRECATED ** Due to non responsiveness. Use <span class="highlight">lightbox-dialog</span> instead </p>
```

Comment:
The `drawer-dialog` has technically been deprecated for a while. I thought we were completely removing the component.

Reply:
No, we cannot. 
DS says we need to keep it in because we do not have a component that is similar. 
Either way, I think this would cause too much backlash, we need to keep it in for now. 

We can remove it from docs if its better.